Improve Forms section of the editor

Ayman161803 opened the pull request:
After changes made, the editor will be as found at https://inspiring-muffin-68f087.netlify.app/
### Final Forms UI:
<img width="1278" alt="image" src="https://user-images.githubusercontent.com/76606666/220179391-5200d860-6f87-492f-8d73-86267141bac8.png">

Closes #4

Goals completed :
---

- Addition of namespaces, declarations through form

https://user-images.githubusercontent.com/76606666/208243336-afd0085a-d6b3-4024-8431-5588d0ddc8c7.mov




- Enable access to ``default``, ``IsOptional``,``regex`` part of an attribute through the form itself

https://user-images.githubusercontent.com/76606666/208243299-33eaa753-745a-4e86-b5f9-cad53de15ac2.mov

- Automatic changes to imports when the name of a namespace being imported is changed through forms.


https://user-images.githubusercontent.com/76606666/208243409-b18b4b49-f3b1-4f07-8db2-ea96dc45d4c8.mov

-  Acess to ``range`` attribute through forms

https://user-images.githubusercontent.com/76606666/208243519-7f1b6c9e-5493-49a8-b03c-d5feb15ea3fa.mov

- Delete button to delete namespace, property, declaration
<img width="1280" alt="image" src="https://user-images.githubusercontent.com/76606666/220169212-00d088cc-f1ed-4bb5-ba82-fd8277b6af31.png">

<img width="1280" alt="image" src="https://user-images.githubusercontent.com/76606666/220169240-cc395dee-4a32-4d15-9210-adea7183532c.png">

<img width="1280" alt="image" src="https://user-images.githubusercontent.com/76606666/220169326-c65b3879-5288-4f45-95b0-62492c247794.png">



- Export .cto models in other targets 


https://user-images.githubusercontent.com/76606666/220169005-f6fd186a-97e7-4ed0-8968-6ab54aeef132.mp4

mttrbrts commented:
Can you add a button/icon to allow a user to close a dialog box without submitting the form, please? Right now, I can't implicitly close it by clicking the background.

Also can you fix the DCO sign-off, please?
https://github.com/accordproject/lab-concerto-editor-web/pull/5/checks?check_run_id=10378671773

dselman commented:
If you sign the DCO and I can merge this...
https://github.com/accordproject/lab-concerto-editor-web/pull/5/checks?check_run_id=11467642723

Ayman161803 commented:
Fixed DCO issues and added close button(Screenshot below) @mttrbrts  @dselman 


<img width="1280" alt="image" src="https://user-images.githubusercontent.com/76606666/220181294-0e490f8e-0e47-40bf-b05d-993c4f55ec8e.png">
